TravelBuddies/TravelBuddies/Models/Comment.m

```diff
@@ -0,0 +1,32 @@
+//
+//  Comment.m
+//  TravelBuddies
+//
+//  Created by Mariana Martinez on 04/08/20.
+//  Copyright © 2020 Mariana Martinez. All rights reserved.
+//
+
+#import "Comment.h"
+
+@implementation Comment
+
+@dynamic author;
+@dynamic comment;
+@dynamic post;
+
++ (nonnull NSString *)parseClassName {
+    return @"Comment";
+}
+
++ (void) postComment: (NSString * _Nullable)comment toPost: (Post * _Nullable)post withCompletion: (PFBooleanResultBlock  _Nullable)completion {
+    
+    Comment *newComment = [Comment new];
+    newComment.author = [PFUser currentUser];
+    newComment.comment = comment;
+    newComment.post = post;
+    
+    [newComment saveInBackgroundWithBlock: completion];
+    
+}
```

Comment:
You could also separate this into an init method and a posting method that handles the errors and completion block. This would allow you do any post processing on the Comment object if you need it